### PR TITLE
fix: create valid (for us) id selectors

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

When creating an action from an event in the live events table. We will pick selectors like "#my-id" or ".one-class.and-another". But we can't actually match those selectors when processing the action 🤦 

Fixing that is hard, but in the meantime we can not create selectors that won't work

## Changes

* generates `[id="blah"]` instead of `#blah`
* ~~generates `[class="first"][class="second"]` instead of `.first.second`~~

## How did you test this code?

added a developer test and created an action locally that matched on ID

### CODE REVIEW

**Review by neilkakkar — APPROVED** [review]
Makes sense!

Did you test locally the second case works? `[class=a][class=b]`

**Comment by pauldambra** [comment]
> Did you test locally the second case works? [class=a][class=b]

Good question! I tested it was a valid CSS selector 🤦 

ruh roh
```
DB::Exception: OptimizedRegularExpression: cannot compile re2: .*?class="rounded"][class="border"][class="p-4".*?([-_a-zA-Z0-9\.:"= ]*?)?($|;|:([^;^\s]*(;|$|\s))), error: invalid character class range: p-4. Look at https://github.com/google/re2/wiki/Syntax for reference.
```

**Comment by neilkakkar** [comment]
🙈

**Comment by pauldambra** [comment]
Oh my! That's all very complex 🤯 

Stripped back to just the working ID case for now

File: frontend/src/lib/actionUtils.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    it('generates a data attr not an #ID', () => {
+        const element = {
+            attr_id: 'tomato',
+        } as ElementType
+
+        const actual = elementToSelector(element, [])
+        expect(actual).toEqual('[id="tomato"]')
+    })
+
+    it('generates an incorrect class selector', () => {
```
Comment (neilkakkar) [inline frontend/src/lib/actionUtils.test.ts:14]: this test 😂 .

LGTM 👍

**Pull request merged**